Tribler/Core/Session.py

```diff
@@ -648,6 +651,41 @@ def create_channel(self, name, description, mode=u'closed'):
         """
         return self.lm.channel_manager.create_channel(name, description, mode)
 
+    @blocking_call_on_reactor_thread
```

Comment:
Then it is being called it from a different thread (the REST ui should be running on the reactor thread). You should figure out what's wrong.